src/main/rx/crsf_protocol.h

```diff
@@ -27,7 +27,9 @@
 #include <stdint.h>
 #include <stdbool.h>
 
-#define CRSF_BAUDRATE           420000
+// Rev7 CRSF docs: UART runs at 400000 baud, 8N1 at 3.0 to 3.3V level.
+// Rev10 CRSF docs: UART runs at 416666 baud, 8N1 at 3.0 to 3.3V level.
+#define CRSF_BAUDRATE           416666
```

Comment:
As alternative for TBS purists we could introduce a build option to apply
```suggestion
// Avoid using with ExpressLRS STM32 receivers.
#ifdef USE_CRSF_OFFICIAL_SPEC
#define CRSF_BAUDRATE           416666
#else
#define CRSF_BAUDRATE           420000
#endif
```

Reply:
@tbs-trappy please review

Reply:
> @tbs-trappy please review

The original 416.6 came from OpenTX dev team and our Crossfire team lead when we were looking for a higher baudrate than the standard 115.2. I think 500 didn't quite work, so we settled on the max that could be achieved at the time, which happened to be this ungodly number.

There are quite a few FCs out there that use and support CRSF which aren't necessarily public knowledge, and they all are using 416.6. some of them are quite sensitive to baudrate changes. Therefore I would suggest to stick with the original 416.6. I never fully understood where 420 came from. We did have a typo in our older docs but that was 400.0. 

The right / "purist" way forward imho would be to stop using 420 on all projects (pun not intended :) ). I don't suggest changing a standard with potential incompatibilities for a whole set of legacy hardware that none of us have any control over for a small set of receivers that "we" do have control over and can change.

Reply:
@haslinghuis suggest you raise another PR to invert the logic.

Reply:
I searched backwards and found that Betaflight has used 420k since 3.5.7 at least.  Since then a lot of ELRS type devices have been built that are set to 420k.  Given that no-one really complained for all those years about us using 420k, and the lack of any serious issues over that time frame, and the now more widespread use of 420k, I personally think we should keep our default at 420k.  For TBS users, especially those with a reason to use 416666, they now have a reasonably simple option at build time.